### PR TITLE
Add 'init' argument

### PR DESCRIPTION
**Pull request by ursi**
This adds an `init` option to the CLI that creates an empty `elm-git.json` for you, so you don't have to remember/look up what the file is called and how it's structured.

### CODE REVIEW

**Comment by ursi** [comment]
This adds an `init` option to the CLI that creates an empty `elm-git.json` for you, so you don't have to remember/look up what the file is called and how it's structured.

File: index.js [hunk 1]
```diff
@@ -13,10 +13,38 @@ const storagePath = path.join('elm-stuff', 'gitdeps');
 
 const args = process.argv.slice(2);
 
+const helpMsg =
+`use 'elm-git-install' with no arguments to install the dependencies in you 'elm-git.json' file
```
Comment (adamdicarlo) [inline index.js]: typo: s/you/your/

(I'm not a maintainer, but happened to look at this.)

Comment (ursi) [reply index.js]: Fixed 👍

**Comment by robinheghan** [comment]
Thank you for this! 💯

**Pull request merged**